spec/01-unit/02-conf_loader_spec.lua

```diff
@@ -341,6 +341,34 @@ describe("Configuration loader", function()
           assert.True(helpers.path.isabs(conf.ssl_cert))
           assert.True(helpers.path.isabs(conf.ssl_cert_key))
         end)
+        it("defines ssl_ciphers by default", function()
+          local conf, err = conf_loader(nil, {})
+          assert.is_nil(err)
+          assert.matches(":", conf.ssl_ciphers) -- looks kinda like a cipher suite
+        end)
+        it("errors on invalid ssl_cipher_suite", function()
+          local conf, _, errors = conf_loader(nil, {
+            ssl_cipher_suite = "foo"
+          })
+          assert.equal(1, #errors)
+          assert.matches("Undefined cipher suite foo", errors[1])
+        end)
+        it("overrides ssl_ciphers when ssl_cipher_suite is custom", function()
+          local conf,err = conf_loader(nil, {
+            ssl_cipher_suite = "custom",
+            ssl_ciphers      = "foo:bar",
+          })
+          assert.is_nil(err)
+          assert.equals("foo:bar", conf.ssl_ciphers)
+        end)
+        it("doesn't override ssl_ciphers when undefined", function()
+          local ciphers = require "kong.tools.ciphers"
+          local conf, err = conf_loader(nil, {
+            ssl_cipher_suite = "custom",
+          })
+          assert.is_nil(err)
+          assert.equals(ciphers("modern"), conf.ssl_ciphers)
```

Comment:
Better to use `assert.same()` when comparing tables, at any time except rare exceptions.